hadolint.sh

```diff
@@ -44,7 +44,10 @@ if [ -n "$HADOLINT_OUTPUT" ]; then
 fi
 
 RESULTS="${RESULTS//$'\\n'/''}"
-echo "::set-output name=results::$RESULTS"
+
+echo "results<<EOF" >> $GITHUB_OUTPUT
+echo "${RESULTS}" >> $GITHUB_OUTPUT
+echo "EOF" >> $GITHUB_OUTPUT
```

Comment:
wait does this actually work?

Reply:
Yes I tested in a local non forked copy of the repository: 

The Code Changes: https://github.com/mrdoodles/hadolint-local/pull/3/files 
The Run: https://github.com/mrdoodles/hadolint-local/actions/runs/3431727905/jobs/5720205348

Reply:
Why not just a simple way e.g.
`echo "results=${RESULTS}" >> $GITHUB_OUTPUT` ?
EOF syntax makes sense to use for multiline things, but there is one line `echo`.

Reply:
Basically this PR https://github.com/hadolint/hadolint-action/pull/64/files does it as well

Reply:
@ViacheslavKudinov Because the output actually is a multiline string so it fails  with the simple format which was my original commit that failed and i force pushed over.

You can see the output from using the simple method here: https://github.com/hadolint/hadolint-action/actions/runs/3307229271/jobs/5671320194

Reply:
Got it. Thanks @mrdoodles for clarification.

Reply:
Having said that I have just tried it again and it has passed: https://github.com/mrdoodles/hadolint-local/actions/runs/3443878466/jobs/5745890302

Reply:
The delimeter shouldn't be randomly generated?